aexpect/client.py

```diff
@@ -181,8 +181,9 @@ def __init__(self, command=None, a_id=None, auto_close=False, echo=False,
             # try to find python specific version of aexpect-helper first, then
             # try unversioned
             helper_noversion = 'aexpect-helper'
-            helper_versioned = '{0}-{1.major}.{1.minor}'.format(
-                helper_noversion, sys.version_info)
+            helper_versioned = '{0}-{1}.{2}'.format(helper_noversion,
+                                                    sys.version_info[0],
+                                                    sys.version_info[1])
```

Comment:
Interesting to remove py2.6 testing and then to introduce 2.6 compat fix. Anyway I do prefer 2.6 compatibility (even when not official) so yes to this.